src/plugins/intel_gpu/src/graph/impls/ocl_v2/include/batch_headers/int4_utils.cl

```diff
@@ -0,0 +1,270 @@
+//  Copyright (C) 2023 Intel Corporation
```

Comment:
random spot:
It is little bit not intuitive for me that the kernels_cache & header cl kernels are under impls, while the cl kernels are under kernels_selector directory, and impls and kernels_cache are in the same hierarchy.. 
(also there are header cl kernels under kernels_selector directory) ..

Reply:
The main idea is that backend specific code should be located in `impls/${backend_name}` folder. headers for kernel selector are in kernel selector still (those are duplicated for now to avoid any dependencies between components). Kernel selector shall be removed in the future, so the whole structure of ocl backend will look like ocl_v2 version